### DISCUSSION TITLE
Board securement

### DISCUSSION THREAD

**Discussion by upgrader-dev** [alf45tar/PedalinoMini#513] [Q&A]
Hello guys,

How do you secure the board on the case? Can't see any screw.

**Comment by alf45tar** [comment]
Which board? Which case? Are you looking into PedalinoMini 4/6/8 assembly?

**Reply by upgrader-dev** [reply]
Oups. Yes, I'm looking into PedalinoMini 8 assembly

**Comment by alf45tar** [comment]
The display is stuck in the ceiling, the board is blocked on one side by the USB port and by the support on the other side. No screws are needed for the board once the box is closed.
